gmake fails on FreeBSD

Hi,

i tried to to build ponyc like described in the readme for FreeBSD but I always get the following error message:

```
Linking libponyrt.tests
build/release/libponyrt.a(threads.o): In function `ponyint_thread_create':
src/libponyrt/platform/threads.c:(.text+0xd): undefined reference to `pthread_create'
c++: error: linker command failed with exit code 1 (use -v to see invocation)
gmake: *** [Makefile:697: build/release/libponyrt.tests] Fehler 1
```

I'm running 11.1-RELEASE FreeBSD 11.1-RELEASE with GNU Make 4.2.1

--- a/Makefile
+++ b/Makefile
@@ -465,6 +465,8 @@ ifeq ($(OSTYPE),linux)
 endif
 
 ifeq ($(OSTYPE),freebsd)
+  libponyc.tests.links += libpthread
+  libponyrt.tests.links += libpthread
   libponyc.benchmarks.links += libpthread
   libponyrt.benchmarks.links += libpthread
 endif

